LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Yago
+Copyright (c) 2019 Yago
```

Comment:
Please leave out this change. This should be in a separate MR (or pushed to the main master branch instead).